.circleci/config.yml

```diff
@@ -72,10 +72,6 @@ jobs:
     - run:
         name: Lint
         command: make BUILD_IN_CONTAINER=false lint
-    # fails to run everything first time - see https://github.com/golangci/golangci-lint/issues/866
```

Comment:
We're currently running `golangci-lint` `v1.21.0` (see `build-image/Dockerfile`) which doesn't include the fix. You should upgrade to the latest `v1.24.0` first, build and publish a new `build-image` (replacing it across `.circleci/config.yml` - there are multiple references) before merging this PR.